cli/config.go

```
@@ -198,7 +198,11 @@ func setConfigValue(key string, value string) error {
 	}
 
 	if strings.Contains(key ,"url") {
-		if ok, err := isValidURL(value); !ok {
+		u, err := url.Parse(value)
```

this err is not handled

---

@SammyOina Please resolve this if it is addressed.